apps/molgenis-components/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "molgenis-components",
+  "name": "@molgenis/molgenis-components",
   "version": "0.0.0",
+  "private": true,
```

Comment:
why this change please add to commit comment

Reply:
renamed back